c/driver/postgres/CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ add_arrow_lib(adbc_driver_postgres
               type.cc
               OUTPUTS
               ADBC_LIBRARIES
+              CMAKE_PACKAGE_NAME
+              AdbcDriverPostgres
```

Comment:
How about using "PostgreSQL" instead of "Postgres"?
"Postgres" is old name of "PostgreSQL" and the official site uses "PostgreSQL" not "Postgres": https://www.postgresql.org/

Reply:
Oops, thanks for checking. I filed https://github.com/apache/arrow-adbc/issues/234 and will update things in general here.